lib/smart_proxy_dhcp_infoblox/fixed_address_crud.rb

```diff
@@ -51,6 +51,7 @@ def find_host_and_name_by_ip(ip_address)
 
     def build_host(options)
       host = ::Infoblox::Fixedaddress.new(:connection => @connection)
+      extra_options = [{:name => 'host-name', :value => options[:hostname]}]
```

Comment:
```suggestion
      options.merge! {:name => 'host-name', :value => options[:hostname]}
```

Reply:
This is a poor suggestion since it achieves something completely different. `options` is never passed to `host.options`.